CMakeLists.txt

```diff
@@ -154,15 +158,18 @@ option(ENABLE_BUNDLED_JSONC "Force use of a vendored copy of JSON-C" False)
 option(ENABLE_BUNDLED_YAML "Force use of a vendored copy of libyaml" False)
 option(ENABLE_BUNDLED_PROTOBUF "Use a vendored copy of protobuf" False)
 
+# SSL
+option(ENABLE_WOLFSSL "Force use of WolfSSL instead OpenSSL" False)
+
```

Comment:
Duplicate option definition.

Reply:
I preferred to use `if defined` everywhere, because there are places where I tested `both`, and I also thought it was more clean than to use `ifdef ... elif defined()` in some places. :thinking: